addon/components/ember-th/component.js

```diff
@@ -162,6 +170,13 @@ export default class EmberTh extends BaseTableCell {
     }
   }
 
+  contextMenu(event) {
+    this.sendAction('onContextMenu', {
```

Comment:
why are we sending an object w/ the action as oppose to just sending an `event`?

Reply:
Copy/paste and a change of mind while testing (I debated sending part of the API/meta) 😄 

Good catch

Reply:
@twokul fixed, ready for review